### PR TITLE
chore(skip-release): Prepare for 2.0.0

### PR DESCRIPTION
**Pull request by angelozerr**
chore(skip-release): Prepare for 2.0.0

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -6,6 +6,17 @@
     </vendor>
 
     <change-notes><![CDATA[
+    <h3>2.0.0</h3>
+    <ul>
+      <li>Delegate "LSP support" for Microprofile / Quarkus / Qute with <a href="https://github.com/redhat-developer/lsp4ij">LSP4IJ</a></li>
```
Comment (fbricon) [inline src/main/resources/META-INF/plugin.xml:11]: s/with/to

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -6,6 +6,17 @@
...
+      <li>Workspace symbols support for JAX-RS endpoints </li>
+      <li>Resolve system properties/environment variables while browsing the application.properties values</li>
```
Comment (fbricon) [inline src/main/resources/META-INF/plugin.xml:13]: ... while browsing application.properties ...

File: src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (angelozerr) [reply src/main/resources/META-INF/plugin.xml]: fixed.

File: src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (angelozerr) [reply src/main/resources/META-INF/plugin.xml]: fixed

**Pull request merged**